src/sections/account/Account.tsx

```diff
@@ -0,0 +1,58 @@
+import { useTranslation } from 'react-i18next'
+import { useSearchParams } from 'react-router-dom'
+import { Tabs } from '@iqss/dataverse-design-system'
+import { useLoading } from '../loading/LoadingContext'
+import { AccountHelper } from './AccountHelper'
+import { ApiTokenSection } from './api-token-section/ApiTokenSection'
+import { BreadcrumbsGenerator } from '../shared/hierarchy/BreadcrumbsGenerator'
+import { UpwardHierarchyNodeMother } from '../../../tests/component/shared/hierarchy/domain/models/UpwardHierarchyNodeMother'
+import styles from './Account.module.scss'
+import { useEffect } from 'react'
+
+const tabsKeys = AccountHelper.ACCOUNT_PANEL_TABS_KEYS
+
+export const Account = () => {
+  const { t } = useTranslation('account')
+  const { setIsLoading } = useLoading()
+  const [searchParams] = useSearchParams()
+  const defaultActiveTabKey = AccountHelper.defineSelectedTabKey(searchParams)
+
+  const rootHierarchy = UpwardHierarchyNodeMother.createCollection({
+    name: 'Root',
+    id: 'root'
+  })
+
+  useEffect(() => {
+    setIsLoading(false)
+  }, [setIsLoading])
+
+  return (
+    <section>
+      <BreadcrumbsGenerator hierarchy={rootHierarchy} withActionItem actionItemText="Account" />
+
+      <header>
+        <h1>{t('pageTitle')}</h1>
```

Comment:
The page title in JSF includes the name of the root collection, but I don't think we need it, because of the breadcrumbs. What do you think?
![Screenshot 2024-09-12 at 10 19 10 AM](https://github.com/user-attachments/assets/e7341a47-bca4-44fc-8dd3-4ad28b8ffe44)

Reply:
Yes, it seemed a bit repetitive, that's why I didn't add it, I think we can leave it for now.

Reply:
Yes, I agree! just wanted confirm